src/Explorer/SplashScreen/SplashScreen.tsx

```diff
@@ -9,7 +9,7 @@ import {
   Stack,
   TeachingBubble,
   TeachingBubbleContent,
-  Text,
+  Text
```

Comment:
You need to add the comma back otherwise the build would fail at the format step.